articles/storage/blobs/storage-dotnet-how-to-use-blobs.md

```diff
@@ -20,385 +20,370 @@ ms.author: tamram
-[!INCLUDE [storage-container-naming-rules-include](../../../includes/storage-container-naming-rules-include.md)]
-
-This example shows how to create a container if it does not already exist:
-
-```csharp
-// Retrieve storage account from connection string.
-CloudStorageAccount storageAccount = CloudStorageAccount.Parse(
-    CloudConfigurationManager.GetSetting("StorageConnectionString"));
-
-// Create the blob client.
-CloudBlobClient blobClient = storageAccount.CreateCloudBlobClient();
-
-// Retrieve a reference to a container.
-CloudBlobContainer container = blobClient.GetContainerReference("mycontainer");
-
-// Create the container if it doesn't already exist.
-container.CreateIfNotExists();
+http://localhost:58673/api/blobs
 ```
 
-By default, the new container is private, meaning that you must specify your storage access key to download blobs from this container. If you want to make the files within the container available to everyone, you can set the container to be public using the following code:
+You should see a response that is similar to the following (if viewed as JSON):
 
-```csharp
-container.SetPermissions(
-    new BlobContainerPermissions { PublicAccess = BlobContainerPublicAccessType.Blob });
+```
+[
+    "Block blob with name 'Awesome-Mountain-River-Wallpaper.jpg', content type 'application/octet-stream', size '1465150', and URI 'http://127.0.0.1:10000/devstoreaccount1/quickstart/Awesome-Mountain-River-Wallpaper.jpg'",
+    "Block blob with name 'Beautiful Stream Desktop Wallpapers (2).jpg', content type 'application/octet-stream', size '382544', and URI 'http://127.0.0.1:10000/devstoreaccount1/quickstart/Beautiful Stream Desktop Wallpapers (2).jpg'",
+    "Block blob with name 'Forest-river-wallpaper-Hd.jpg', content type 'application/octet-stream', size '2690689', and URI 'http://127.0.0.1:10000/devstoreaccount1/quickstart/Forest-river-wallpaper-Hd.jpg'",
+    "Block blob with name 'Grand-Canyon-Colorado-River.jpg', content type 'application/octet-stream', size '261788', and URI 'http://127.0.0.1:10000/devstoreaccount1/quickstart/Grand-Canyon-Colorado-River.jpg'",
+    "Block blob with name 'Smith-River.jpg', content type 'application/octet-stream', size '519207', and URI 'http://127.0.0.1:10000/devstoreaccount1/quickstart/Smith-River.jpg'",
+    "Block blob with name 'grand-canyon-colorado-river-1.jpg', content type 'application/octet-stream', size '232356', and URI 'http://127.0.0.1:10000/devstoreaccount1/quickstart/grand-canyon-colorado-river-1.jpg'",
+    "Block blob with name 'ws_Columbia_Mountain_River_Grass_1366x768.jpg', content type 'application/octet-stream', size '450769', and URI 'http://127.0.0.1:10000/devstoreaccount1/quickstart/ws_Columbia_Mountain_River_Grass_1366x768.jpg'"
+]
 ```
 
-Anyone on the Internet can see blobs in a public container. However, you can modify or delete them only if you have the appropriate account access key or a shared access signature.
+## Basic blob operations
```

Comment:
Do we list somewhere what a blob is? We are saying "blob" everywhere, but we seem to upload a file? image? byte[]?

Reply:
We link to the top. I think in this case, most people who land on this article (_TONS_ from organic search) already sort of know what blob storage is and they just want to see code for how to interact with it.l